tests/test_query_params.py

```diff
@@ -414,6 +418,37 @@ def test_multiple_form_keys_as_list(self):
         # There are three 'cat' keys; order is preserved.
         self.assertEqual(req.get_param_as_list('cat'), ['6', '5', '4'])
 
+    def test_get_date_valid(self):
+        date_value = "20150420"
+        query_string = "thedate={0}".format(date_value)
+        format_string = "%Y%m%d"
```

Comment:
Would you mind adding a test that doesn't pass a format string (uses the default)? That way, if we ever accidentally mess up the default string so it is different than what we've documented (and what the user expects), we'll catch that with a test?